funcx_endpoint/tests/integration/funcx_endpoint/endpoint/test_endpoint_manager.py

```diff
@@ -30,71 +31,70 @@ def test_setup_teardown(self, fs):
         # Code that will run before your test, for example:
 
         funcx_dir = f"{os.getcwd()}"
-        config_dir = os.path.join(funcx_dir, "mock_endpoint")
-        assert not os.path.exists(config_dir)
+        config_dir = pathlib.Path(funcx_dir) / "mock_endpoint"
+        assert not config_dir.exists()
 
         # pyfakefs will take care of newly created files, not existing config
         fs.add_real_file(default_config.__file__)
 
         # A test function will be run at this point
         yield
         # Code that will run after your test, for example:
-        shutil.rmtree(config_dir)
+        # shutil.rmtree(config_dir)
```

Comment:
Is this intentional?

Reply:
Ah; no, but not in the way you think: I meant to _remove_ that line.  We're already within a `pyfakefs` context.  Thank for catching that.

Reply:
I think I'll remove this as part of my next PR.